<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOpenUrl</class>
 <widget class="QDialog" name="DialogOpenUrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打开URL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MediaPlayer.qrc">
    <normaloff>:/MediaPlayer/Resources/MediaPlayer.ico</normaloff>:/MediaPlayer/Resources/MediaPlayer.ico</iconset>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>输入网络上的url地址或路径，支持http https</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>例如：rtmp://live.hkstv.hk.lxdns.com/live/hks2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_url">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>rtmp://live.hkstv.hk.lxdns.com/live/hks2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>URL:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>123</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>清除记录</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_url">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MediaPlayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox_url</sender>
   <signal>accepted()</signal>
   <receiver>DialogOpenUrl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_url</sender>
   <signal>rejected()</signal>
   <receiver>DialogOpenUrl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
